CTP/doc/shell/shell_guide.md

```diff
@@ -0,0 +1,667 @@
+# Shell Test Guide
+# 1 Test Introduction
+Shell test is an important test suit in cubrid test.  
+It contains almost all the feature test and some performance test which cannot be test by sql test or other test suits.  
+
+# 2 Tools Introduction
+CTP is the only test tool which is used in shell test.   
+Source URL: [https://github.com/CUBRID/cubrid-testtools](https://github.com/CUBRID/cubrid-testtools)
+
+# 3 Test Deployments
+## 3.1 create and set users  

```

Comment:
make 3.1 ~ 3.4 indent.

8.1 ... is same.

Reply:
??